test/poa_agent/transfers/mnesia_test.exs

```diff
@@ -0,0 +1,53 @@
+defmodule POAAgent.Plugins.Transfers.DB.MnesiaTest do
+  use ExUnit.Case
+
+  alias POAAgent.Entity.System.Metric
+  alias POAAgent.Plugins.Transfers.DB.Mnesia
+
+  test "sending data to mnesia" do
+    args = %{name: :metrics_transfer, args: [table_name: :metrics,
+                                             fields: [:timestamp, :os_type,
+                                                      :unix_process, :cpu_util,
+                                                      :disk_used, :memsup]]}
+
+    {:ok, _pid} = Mnesia.start_link(args)
+
+    :ok = send_to_transfer(:metrics_transfer, :my_metrics, data_message())
+    Process.sleep(5000)
+
+    read_data = fn -> :mnesia.read({:metrics, 101}) end
+
+    assert {:atomic, [{:metrics, 101, {:unix, :linux}, 101, 10.1, 10,
```

Comment:
you also must check that this entry doesn't exist before send the data to the transfer